src/jumbBox.js

```diff
@@ -1,86 +1,101 @@
-export default class JumbBox {
-    root = null;
+export class JumbBox {
+    rootBox = null;
+    descriptionBox = null;
 
     constructor(jumbBox) {
-        this.root = jumbBox;
+        this.rootBox = jumbBox;
+        this.descriptionBox = this.getDescriptionBox(this.rootBox);
     }
 
-    resolveUri(uri) {
-        let prefix = "self#jumbf="
+    resolveContainerBoxByUri(uri) {
+        let parts = uri.split("/");
+        return this.resolveBoxByPath(this.rootBox, parts);
+    }
 
-        if (!uri || !uri.startsWith(prefix)) {
-            return null;
-        }
+    resolveContainerBoxesByUuid(uuid) {
+        return this.resolveAllBoxes(this.rootBox, (descBox) => descBox.content.uuidString == uuid);
```

Comment:
Probably OK in the current case but generally we should avoid using loose equality (==) because it may lead to unexpected behaviors in JavaScript. Try to use strict equality (===) instead.